### PR TITLE
Updated TODO for Ironic installation

### PR DESCRIPTION
**Pull request by SafeEHA**
Under 4.1 Install cluster-api metal3 provider [TODO](https://book.metal3.io/capm3/TODO) returns an error when clicked so i found the correct [link](https://book.metal3.io/ironic/ironic_installation.html) and updated it

### CODE REVIEW

File: docs/user-guide/src/capm3/installation_guide.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ install them yourself.
 
 1. Install `clusterctl`, refer to Cluster API [book](https://cluster-api.sigs.k8s.io/user/quick-start.html#install-clusterctl) for installation instructions.
 1. Install `kustomize`, refer to official instructions [here](https://kubectl.docs.kubernetes.io/installation/kustomize/).
-1. Install Ironic, refer to [TODO](TODO).
+1. Install Ironic, refer to [TODO](https://book.metal3.io/ironic/ironic_installation.html).
```
Comment (furkatgofurov7) [inline docs/user-guide/src/capm3/installation_guide.md:14]:
```suggestion
1. Install Ironic, refer to [this page](https://book.metal3.io/ironic/ironic_installation.html).
```

**Review by furkatgofurov7 — CHANGES_REQUESTED** [review]
/ok-to-test

Can you also please squash your commits?

**Comment by SafeEHA** [comment]
Yes, I would do so now
Thanks for your suggestion

Please I also noticed that 2.1 Install baremetal operator is not accessible
Do you by chance know whats the issue?

**Comment by dtantsur** [comment]
Seems not squashed still. You can use `git rebase -i` or just re-create the commit.

**Comment by SafeEHA** [comment]
I might have to re-create the commit, for some reason I can't see my commit when I `git rebase -i` but I can see others

**Comment by tuminoid** [comment]
> I might have to re-create the commit, for some reason I can't see my commit when I `git rebase -i` but I can see others

`git rebase -i HEAD~5` should include all your 4 commits, and you can squash them.

**Pull request closed without merging**